src/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "svelte-collapse" {
+  import type { ActionReturn } from "svelte/types/runtime/action";
+
+  interface CollapseParams {
```

Comment:
Should this also be `export`'ed ?

Reply:
Hm, I don't know typescript very well, but I don't think so. I don't need to import ActionReturn or HTMLElement either when I want to _use_ functions later on (just the arguments I pass in/ what I do with the return value has to match).